### DISCUSSION TITLE
suspend for reboot?

### DISCUSSION THREAD

**Discussion by syadnom** [ericaltendorf/plotman#465] [General]
I can't seem to find any mention of this.  Is it possible to suspend a plot, reboot the machine, then resume that plot?

**Comment by Purple-Tentacle** [comment]
No, a reboot destroys the progress. That is a limitation of chia and not plotman.

**Reply by syadnom** [reply]
I figured as much but I thought maybe you knew some trick.  Thanks.

**Reply by altendky** [reply]
There are requests to Chia for this.  Take a peek over there.  I think there were some theories at least for handling it at the end of phase 1.  The closest I have come to 'suspending' plots is using hibernation.  Depending what you need, that may suffice.  For example, moving to a different power outlet or such.

**Reply by syadnom** [reply]
I need to upgrade the hardware, specifically add an LSI HBA, I'm not sure how resuming from hybernate would handle that.  I was hoping for a quick fix but I think I'll just have to let my near complete plots finish and power down :/

**Reply by altendky** [reply]
I am fairly curious about hardware swaps while hibernated myself...  but i haven't tested anything yet nor talked to a Linux group.  Let me know if you run across any info.

**Reply by syadnom** [reply]
I believe linux has all the basics down.  Even CPU hot swap is available in kernel on some platforms.  Not sure if this requires some signals from the chipset though, kernel might implode.
